src/browser-polyfill.js

```diff
@@ -6,7 +6,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 "use strict";
 
-if (typeof browser === "undefined") {
+if (typeof window.browser === "undefined") {
```

Comment:
Quoting myself from https://github.com/mozilla/webextension-polyfill/issues/68#issuecomment-388543539:
> I personally prefer using:
> ```js
> if (!("browser" in window)) {
>     // [...]
> }
> ```
> because using `window.browser` directly when testing if it exists tends to print dereferencing an `undefined` reference warnings in the console.